src/structures/IntegrationApplication.js

```diff
@@ -15,6 +15,48 @@ class IntegrationApplication extends Application {
      * @type {?User}
      */
     this.bot = data.bot ? this.client.users.add(data.bot) : this.bot ?? null;
+
+    /**
+     * The url of the app's terms of service
+     * @type {?string}
+     */
+    this.termsOfServiceURL = data.terms_of_service_url ?? this.termsOfServiceURL ?? null;
+
+    /**
+     * The url of the app's privacy policy
+     * @type {?string}
+     */
+    this.privacyPolicyURL = data.privacy_policy_url ?? this.privacyPolicyURL ?? null;
+
+    /**
+     * The Array of RPC origin urls
+     * @type {string[]}
+     */
+    this.rpcOrigins = data.rpc_origins ?? this.rpcOrigins ?? [];
+
+    /**
+     * The application summary
+     * @type {?string}
+     */
+    this.summary = data.summary ?? this.summary ?? null;
+
+    /**
+     * Whether the application can be default hooked by the client
```

Comment:
```suggestion
     * Whether the application's game can be default hooked by the client
```

Reply:
Not necessarily going to stick to games, i'd even argue "watch youtube together" is not much of a game, I'll keep the more generic description